README.md

```diff
@@ -1,13 +1,137 @@
-erl_optics
-=====
+# erl_optics
 
-An OTP library
+An OTP application for [optics](https://github.com/RAttab/optics).
 
-### Requirements
+## Requirements
+-----
 
 * Erlang OTP >= 21
+* [optics](https://github.com/RAttab/optics) v1.1.10
 
-Build
+## Building
 -----
-
     $ rebar3 compile
+
+## Usage
+-----
+
+For now, the only working mode is Carbon.
+
+need env :
```

Comment:
Mention the name of the file, where it needs to be stored for erl-optics to pick it up.

Reply:
This kinda depends on how the end user configures Erlang releases. In our case we'd put it in `env.config` in Salt, but other people do other things, perhaps even setting things manually in the invocation of `erl` via `-erl_optics foo bar`